test/functional/announcements_controller_test.rb

```diff
@@ -231,55 +260,6 @@ def refute_select(hash)
     end
   end
 
-  view_test "index generates an atom feed with entries for announcements matching the current filter" do
-    with_stubbed_rummager(@rummager, true) do
-      location = create(:world_location, name: "beetlejuice")
-      create(:published_news_story, world_locations: [location], first_published_at: 1.week.ago)
-
-      @rummager.expects(:search).returns(search_results_from_rummager)
-
-      processed_rummager_documents = search_results_from_rummager['results'].map do |result|
-        RummagerDocumentPresenter.new(result)
-      end
-
-      get :index, params: { world_locations: [location.to_param] }, format: :atom
-
-      assert_select_atom_feed do
-        assert_select_atom_entries(processed_rummager_documents, false)
-      end
-    end
-  end
-
-  view_test "index generates an atom feed with the legacy announcement_type_option param set" do
-    with_stubbed_rummager(@rummager, true) do
-      create(:published_news_story, first_published_at: 1.week.ago)
-
-      @rummager.expects(:search).returns(search_results_from_rummager)
-
-      processed_rummager_documents = search_results_from_rummager['results'].map do |result|
-        RummagerDocumentPresenter.new(result)
-      end
-
-      get :index, params: { announcement_type_option: 'news-stories' }, format: :atom
```

Comment:
What happens if you set the `locale: :fr` here?

Reply:
It doesn't seem to like it, I've passed in the `locale: :fr` as one of the params and the test seems to fail

Reply:
I think this is because the localised finders don't really have any parameters to set, so they can't inherit any which is what this test is looking at.